package/MDAnalysis/analysis/rdf.py

```diff
@@ -229,6 +234,13 @@ def __init__(self,
         self.rdf_settings = {'bins': nbins,
                              'range': range}
         self._exclusion_block = exclusion_block
+        if exclude_same is not None and exclude_same not in ['residue', 'segment', 'chain']:
```

Comment:
Can you use `exclusion_block` and `exclude same` and get something that makes sense? If not throw an error. if you try to use both.

Reply:
Dissallowed this since it is quite confusing even if it technically works